src/com/company/HW_less_2.java

```diff
@@ -0,0 +1,57 @@
+package com.company;
+
+public class HW_less_2 {
+
+    public static void main(String[] args) {
+
+        gapNum(5,61);
+        numType(-6);
+        numType_2(0);
+        print("Слово которое печатать ", 4);
+        checkLeapYear(2130);
+
+    }
+
+    public static void gapNum(int a, int b){
+        int sum = a+b;
+        // можно конечно и без объявление переменной sum, но сделал так :)
+
+        // if(sum >= 10 && sum <= 20) System.out.println("true");
+        // else System.out.println("false");
+
+        System.out.println(sum >= 10 && sum <= 20);
+    }
+
+    public static void numType(int b){
+
+        if(b >= 0) System.out.println("Число положительно");
+        else System.out.println("Число отричательно");
+    }
+
+    public static void numType_2(int num){
```

Comment:
```suggestion
    public static void numType2(int num){
```
при именовании нижнее подчеркивание в классах/методах/переменных не приветствуется